src/frontend/src/pages/ProjectDetailPage/ProjectForm/ProjectForm.tsx

```diff
@@ -172,14 +150,22 @@ const ProjectFormContainer: React.FC<ProjectFormContainerProps> = ({
         title={project ? `${wbsPipe(project.wbsNum)} - ${project.name}` : 'New Project'}
         previousPages={[{ name: 'Projects', route: routes.PROJECTS }]}
         headerRight={
-          <Box textAlign="right">
+          <Stack textAlign="right" direction="row" justifyContent={'flex-end'}>
             <NERFailButton variant="contained" onClick={exitEditMode} sx={{ mx: 1 }}>
               Cancel
             </NERFailButton>
-            <NERSuccessButton variant="contained" onClick={(event) => handleSubmit} type="submit" sx={{ mx: 1 }}>
-              Submit
+            <NERSuccessButton
+              variant="contained"
+              onClick={onSubmitChangeRequest ? () => setIsModalOpen(true) : (event) => handleSubmit}
+              sx={{ mx: 1 }}
+            >
+              {onSubmitChangeRequest ? 'Generate Change Request' : 'Submit'}
```

Comment:
make sure to mimic the style of the way we have the work package form buttons set up, 3 buttons with the info icon on the left. They should be appropriately disabled if a change request is or is not entered (see behavior in work package form)